lib/Rex/Pkg/FreeBSD.pm

```diff
@@ -56,10 +58,14 @@ sub bulk_install {
 sub remove {
   my ( $self, $pkg ) = @_;
 
-  my ($pkg_found) = grep { $_->{"name"} eq "$pkg" } $self->get_installed();
-  my $pkg_version = $pkg_found->{"version"};
+  my $pkg_version = '';
 
-  return $self->SUPER::remove("$pkg-$pkg_version");
+  if ( $pkg !~ /\*/ ) {
+    my ($pkg_found) = grep { $_->{"name"} eq "$pkg" } $self->get_installed();
+    $pkg_version = '-' . $pkg_found->{"version"};
+  }
+
+  return $self->SUPER::remove("$pkg$pkg_version");
```

Comment:
So if it's not a package glob, prefix the version string with `-`, otherwise leave it as it is.

Just for my education, could you provide an example about how package and package glob versions look like on FreeBSD, please?

Reply:
This in testing resulted in trying to remove 'foo*-'.  The - at the end would mean it will never match anything.